package.json

```diff
@@ -21,10 +21,10 @@
   "productName": "Hathor Wallet",
   "description": "Light wallet for Hathor Network",
   "author": "Hathor Labs <contact@hathor.network> (https://hathor.network/)",
-  "version": "0.10.0",
+  "version": "0.11.0",
   "private": true,
   "dependencies": {
-    "@hathor/wallet-lib": "^0.7.0",
+    "@hathor/wallet-lib": "^0.8.0",
     "@sentry/electron": "^0.17.0",
```

Comment:
Should we remove Sentry?

Reply:
I think it's good to remove I just don't know if we should do it now, or in the next release

Reply:
My opinion is that we should remove for the mainnet wallet.

Reply:
I think I feel the same. I was just thinking what could go wrong removing it now but I guess it's not complicated. I could remove it.

@msbrogli what do you think?

Reply:
I don't we have to disable it by default and skip the initial screen requesting to enable it. But we should keep the choice in the settings, so, if any user has a problem, they can enable it to let us know the details.

Reply:
Agreed.

Reply:
Done